ui/src/organizations/components/CollectorList.tsx

```diff
@@ -28,7 +28,8 @@ export default class CollectorList extends PureComponent<Props> {
         <IndexList>
           <IndexList.Header>
             <IndexList.HeaderCell columnName="Name" width="50%" />
-            <IndexList.HeaderCell columnName="Bucket" width="50%" />
+            <IndexList.HeaderCell columnName="Bucket" width="25%" />
+            <IndexList.HeaderCell columnName="" width="25%" />
```

Comment:
what's this column?

Reply:
The rows have a unnamed third column (which is usually reserved for actions), just making sure there are equal number of columns in the header as the actual table

Reply:
👍